### ISSUE TITLE
Parameter to define output format

### ISSUE THREAD

**Issue by CharlyAlonso** [lovasoa/dezoomify-rs#148]
Hi,

It would be very interesting to have a parameter that defines the output format. For instance:

-f [png; jpg; iiif]

With this proposal we can facilitate the download of, for example, Google Art, preserving the default name.

My proposal would look like this (with an example):

dezoomify-rs.exe -r 3 -l -f iiif "https://artsandculture.google.com/asset/the-birth-of-venus-sandro-botticelli/MQEeq50LABEBVg"

Thanks for everything and best regards

**Comment by lovasoa** [comment]
Yes, that would be a nice feature! Would you be interested in implementing it?

**Comment by CharlyAlonso** [comment]
Hello there,
I do not quite understand very well with the implantation ???
I tell you. Suddenly I discovered the wonderful functionality of your wonderful tool in the .iiif format (too bad of me for not having read everything before the instructions).
I have started again with the download of the Gigapixel images that I already had downloaded in other formats and the ones that I am discovering in the ocean of the Internet, new or that I did not know.
Why in iiif format? Because it allows, as you well know, to skip the limit of formats (.jpg, png, tif, etc) and it is an exact image of what is published on the Internet.
Likewise, once the .iiif folder has been generated locally, you can always run "dezoomify-rs info.json" to get a file in the format you want.
I don't know, manias !!!!!
Thank you so much for your incredible work.

**Comment by lovasoa** [comment]
Hey, you are not bothering at all, sorry for the tone ! I meant it; I'll leave this issue opened and implement it when I get to it.

**Comment by lovasoa** [comment]
This is a good feature suggestion

**Comment by SmithJustinL** [comment]
I'd add to the suggestion that the file be as high-quality as the normal Dezoomify files are. As it stands, the downloaded file is much smaller in size and lower in quality than if one could download it through the normal Dezoomify extension. A 200 MB file gets turned into less than 20 MB.

**Comment by AlexanderDeLargeBrotherSir** [comment]
This issue was solved eons ago, @CharlyAlonso, please close this.

**Comment by CharlyAlonso** [comment]
Clised
